starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -776,6 +777,20 @@ public static void createVideoMediaCodecBridge(
       }
     }
 
+    if (maxInputSize != -1) {
+      try {
+        mediaFormat.setInteger(MediaFormat.KEY_MAX_INPUT_SIZE, maxInputSize);
+        Log.i(
+            TAG,
+            "KEY_MAX_INPUT_SIZE is "
+                + mediaFormat.getInteger(android.media.MediaFormat.KEY_MAX_INPUT_SIZE)
+                + " after setting it to "
+                + maxInputSize
+                + '.');
+      } catch (Exception e) {
+        Log.w(TAG, "MediaFormat.getInteger(KEY_MAX_INPUT_SIZE) failed with exception: ", e);
```

Comment:
We should use Log.e() as it's an unexpected error.

Reply:
Changed all 3 Log.w() to Log.e().